build/scripts/create-perftests.ps1

```diff
@@ -0,0 +1,32 @@
+# Create the PerfTests directory under Binaries\$(Configuration).  There are still a number
```

Comment:
Can you document the dependency (or at least explain it here?) I'm curious as I'm looking to delete some of that.